alpine/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM docker:19.03.11 as static-docker-source
+FROM docker:19.03.11
```

Comment:
Can you provide details on what tests you performed to ensure that the container worked properly after this change, for both standard and Docker-in-Docker use cases?